05/app.js

```diff
@@ -9,6 +9,57 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+//na spotkanku mam kilka pytań odośnie tego zadania :). Podejrzewam ze i tak do poprawy :<
+
+const linkElement = document.querySelectorAll('.link');
+const paragraphElement = document.querySelectorAll('.text');
+let numberP1 = 1;
+let clickParagaphCounter = 1;
+let counter = 1;
+
+const clickCounter = function(){
+
+    if (this.dataset.id === 'p1'){
+        stats.paragraphs.p1 = numberP1++;
```

Comment:
Trzeba wykorzystać takie rozwiązanie `stats.paragraphs[this.dataset.id]`, ale trzeba sprawdzać czy już takie coś istnieje ;)
Nie chce wszystkiego zdradzać :P